exercise/templates/exercise/staff/results.html

```diff
@@ -168,6 +168,13 @@ <h4>{% trans "Summaries" %}</h4>
 			</ul>
 		</div>
 
+		<!--Alerts-->
+		<div id="alerts" style="margin: 30px">
+		   <div id="ajax-failed-alert" class="alert alert-danger collapse" role="alert">
+  				Data couldn't be fetched. Please check your internet connection.
```

Comment:
For django translations, wrap that to `{% trans "Data couldn't be fetched. Please check your internet connection." %}`. I can add translations after the merge.

Reply:
Marvelous :+1: